code/src/index.css

```diff
@@ -1,13 +1,139 @@
 body {
+  padding: 20px;
+  background-color: #242626;
+  display: flex;
+  justify-content: center;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+/* code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
+} */
+
+h1 {
+  font-family: Helvetica, sans-serif;
+  color: #fff;
+  font-size: 25px;
+  border-bottom: 2px solid rgba(255, 255, 255, 0.1);
+}
+
+a {
+  display: block;
+  font-family: Helvetica, sans-serif;
+  color: #a0a0a0;
+  font-size: 14px;
+  margin-bottom: 5px;
+  text-decoration: none;
+  /* border: 1px solid red; */
+}
+
+a:hover {
+  color: #fff;
+  text-decoration: underline;
+}
+
+.App {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 2em;
+}
+
+.cover-icons-container {
+  position: relative;
+}
+
+.album-card {
+  width: 100%;
+}
+
+.cover-icons-comtainer img {
+  width: 100%;
+}
+
+.cover-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 1em;
+  /* width: 100%; */
+}
+
+.cover-icons-container:hover .icons-container {
+  opacity: 1;
+  transition: 0.5s ease;
+}
+
+.cover-icons-container:hover .cover-container {
+  opacity: 0.4;
+  transition: 0.5s ease;
+}
+
+.icons-container {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  justify-items: center;
+  align-items: center;
+  opacity: 0;
+}
+
+/* invert color svg icon */
+.icon {
+  filter: brightness(0) invert(1);
+}
+
+.heart,
+.dots {
+  width: 40%;
+}
+
+.play {
+  width: 70%;
+}
+
+.play:hover {
+  width: 100%;
+  transition: 0.5s ease;
+}
+
+.album-title {
+  color: #fff;
+  font-weight: 700;
+}
+
+.artist-name-container > :not(:last-child):not(:first-child):before {
+  content: ', ';
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+.artist-name-container > :not(:first-child):last-child:before {
+  content: ' & ';
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+/* FOR TABLET */
+@media (min-width: 667px) {
+  .App {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+/* FOR DESKTOP */
+@media (min-width: 1024px) {
+  .App {
+    grid-template-columns: repeat(4, 1fr);
```

Comment:
If  you had problems that the columns in the grid are to big, just make the columns slightly smaller, so instead of 50%-50% maybe do 49%-49%, or four times 24%.